config-windows.md

```diff
@@ -0,0 +1,99 @@
+# Windows-specific Container Configuration
+
+This document describes the schema for the [Windows-specific section](config.md#platform-specific-configuration) of the [container configuration](config.md).
+The Windows container specification uses APIs provided by the Windows Host Compute Service (HCS) to fulfill the spec.
+
+You can configure a container's resource limits via the `resources` field of the Windows configuration.
+Do not specify `resources` unless required.
+
+
+#### Memory
+
+`memory` is an OPTIONAL configuration for the container's memory usage.
+
+The following parameters can be specified:
+
+* **`limit`** *(uint64, optional)* - sets limit of memory usage in bytes
+
+* **`reservation`** *(uint64, optional)* - sets soft limit of memory usage in bytes
```

Comment:
Linux has [resource limits with soft and hard settings][1]:

> The  soft  limit  is  the value that the kernel enforces for the corresponding resource.  The hard limit acts  as  a  ceiling for the soft limit:  an  unprivileged process may set only its soft limit to a value in the range from 0 up to the hard limit, and (irreversibly) lower its hard limit.  A privileged process (under Linux: one with the `CAP_SYS_RESOURCE` capability) may make arbitrary changes to either limit value.

So when you say “soft limit”, that's what I'm thinking.  However, “reservation” sounds more like “I need at least this much”.  Is this setting a minimum amount required by the container?  Or an initial limit on the amount that the container can consume?

[1]: http://man7.org/linux/man-pages/man2/getrlimit.2.html